WIP-CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - The `clusterDns` config variable now matches Kubespray defaults.
   Using the wrong value causes node-local-dns to not be used.
+- Kube-prometheus-stack are now being upgraded from 12.8.0 to 16.6.1 to fix dashboard errors.
```

Comment:
It would be nice if this also said which versions of prometheus and grafana that we now use.

Reply:
Hmm! Lets see what i can do! :+1:

Reply:
The user-grafana have now been upgraded to version 8.0.1, sharing the same major-version as the ops one! :+1: